### PR TITLE
Refine Rebrand cities thank you page

### PR DESCRIPTION
**Pull request by fditrapani**
@markryall this is a PR based off the branch you started. It adds the content and imagery we need for the thank you page. Here's a screenshot of what it should look like:

image

### CODE REVIEW

File: client/my-sites/checkout/checkout-thank-you/index.jsx [hunk 1]
```diff
@@ -244,6 +244,13 @@ const CheckoutThankYou = React.createClass( {
 			/* eslint-enable wpcalypso/jsx-classname-namespace */
 		}
 
+		// Rebrand Cities thanks page
+		if ( isRebrandCitiesSiteUrl( this.props.selectedSite.URL ) && PLAN_BUSINESS === this.props.selectedSite.plan.product_slug ) {
```
Comment (fditrapani) [inline client/my-sites/checkout/checkout-thank-you/index.jsx]: I had to move the thank you page up here. As a result, the logic isn't working right. For testing purposes, I set this to `TRUE` so I could see what it looked like. 

`this.props.selectedSite.URL` passes the full wordpress.com url and it was returning false.

File: client/my-sites/checkout/checkout-thank-you/rebrand-cities-thank-you.jsx [hunk 2]
```diff
@@ -2,30 +2,41 @@
  * External dependencies
  */
 import React, { Component } from 'react';
-import Gridicon from 'gridicons';
 import { localize } from 'i18n-calypso';
+import ThankYouCard from 'components/thank-you-card';
 
 class RebrandCitiesThankYou extends Component {
+	renderLogo() {
+		return <img src="/calypso/images/rebrand-cities/wp-rebrand-cities-logo.svg" />;
```
Comment (fditrapani) [inline client/my-sites/checkout/checkout-thank-you/rebrand-cities-thank-you.jsx]: The SVG was way too big to add here so I included it as an image. This cases the image to "flash" into view after it loads. Any thoughts of another way to do this?

File: client/my-sites/checkout/checkout-thank-you/rebrand-cities-thank-you.jsx [hunk 3]
```diff
@@ -2,30 +2,41 @@
...
-
-						<h2 className="checkout-thank-you__header-text">
-							{ translate( 'We will contact you shortly' ) }
-						</h2>
-					</div>
-				</div>
+			<div className="plan-thank-you-card checkout-thank-you__rebrand-cities">
+				<ThankYouCard
+					name={ translate( 'Rebrand Cities package' ) }
+					price="$299.00"
```
Comment (fditrapani) [inline client/my-sites/checkout/checkout-thank-you/rebrand-cities-thank-you.jsx]: Should we be passing this value dynamically vs hard coding it? Can we show the discounted price if that's what they paid?

File: client/my-sites/checkout/checkout-thank-you/index.jsx (hunk 1 above)
Comment (markryall) [inline client/my-sites/checkout/checkout-thank-you/index.jsx]: ah yeah - that's the wrong site property :/ i'll pull your branch into mine and fix that

File: client/my-sites/checkout/checkout-thank-you/rebrand-cities-thank-you.jsx (hunk 3 above)
Comment (markryall) [inline client/my-sites/checkout/checkout-thank-you/rebrand-cities-thank-you.jsx]: i'm pretty sure the purchase will have this information

**Pull request merged**